src/cmd/list.go

```diff
@@ -240,15 +248,16 @@ func listOutput(images []toolboxImage, containers []toolboxContainer) {
 		fmt.Fprintf(writer, "%s\t%s\t%s\n", "IMAGE ID", "IMAGE NAME", "CREATED")
 
 		for _, image := range images {
-			imageName := "<none>"
-			if len(image.Names) != 0 {
-				imageName = image.Names[0]
+			if len(image.Names) == 0 {
+				image.Names = []string{"<none>"}
 			}
+			for _, imageName := range image.Names {
```

Comment:
Looping over the multiple names of an image here will break the sorting.  Even though the code broke in [commit 2369da5d31830e5c](https://github.com/containers/toolbox/commit/2369da5d31830e5cd3e1a13857a686365875ae61), the intention is to sort the output by the names of the images.

So, we need to flatten the names earlier, possibly in `getImages()`, then sort them, and then list them.

Reply:
Done.  However, this has an impact on the shell completions, so those need to be tested.

Reply:
I tested the shell completions manually and they seem to work.

Since `getImages` is also used by the `rmi` command, I spent some time beefing up the tests for it.  See https://github.com/containers/toolbox/pull/1195